lib/chef/provider/package/pacman.rb

```diff
@@ -27,29 +27,12 @@ class Pacman < Chef::Provider::Package
         provides :package, platform: "arch"
         provides :pacman_package
 
+        use_multipackage_api
```

Comment:
should also use_allow_nils

Reply:
added `allow_nils`. Somehow the CI build failed. I'm unsure why, doesn't seem related to my changes.